### PR TITLE
add test for allow-ariahideapp-prop

### PR DESCRIPTION
**Pull request by umahaea**
- add test for allow-ariahideapp-prop

### CODE REVIEW

File: src/test/java/standAloneTests/ModalTest.java [hunk 1]
```diff
@@ -632,6 +631,32 @@ private void successButtonHandlerTest(String isDisableSuccessBtn, Boolean isSucc
...
+        };
+    }
+
+    @Test(testName = "AriaHideApp Test", dataProvider = "AriaHideApp Test Data", groups = "desktop-regression")
+    private void ariaHideAppTest(String ariaHideApp, Object expAriaHidden, String appElement, boolean expWrapper) throws Exception {
+        String[] detailsPropertiesList = new String[]{"elementId", "modal-target"};
+        String[] propsTextList = new String[]{"initiatingButtonText", "any string", "headerTitle", "Terms n Conditions (basic title)", "closeButtonSRText", "close", "modalSaveButtonText", "save", "modalCancelButtonText", "cancel"};
+        String[] propsPropertiesList = new String[]{"isShown", "true", "ariaHideApp", ariaHideApp, "appElement", appElement, "footerVisible", "true", "children", "React.createElement('p', {}, 'Lorem ipsum dolor sit amet')"};
+        setConfigAndLaunch(detailsPropertiesList, propsTextList, propsPropertiesList);
+        commonUtils.printFileContents(modalJSFilePath);
```
Comment (udhadpa) [inline src/test/java/standAloneTests/ModalTest.java]: do we need this line?

Comment (umahaea) [reply src/test/java/standAloneTests/ModalTest.java]: good catch, I wil remove it

File: src/test/java/standAloneTests/ModalTest.java [hunk 2]
```diff
@@ -632,6 +631,32 @@ private void successButtonHandlerTest(String isDisableSuccessBtn, Boolean isSucc
...
+
+        boolean actWrapper = commonUtils.isElementPresent(modalPgObj.divWrapper);
+        isWrapperPresent = commonUtils.assertValue(actWrapper, expWrapper, "wrapper is present inspite of having ariaHiddenApp prop set");
+
```
Comment (StommePoes) [inline src/test/java/standAloneTests/ModalTest.java]: ariaHiddenApp is used in place of the wrapper? I thought it was used as well as the wrapper.

Comment (umahaea) [reply src/test/java/standAloneTests/ModalTest.java]: The first check is for the 'wrapper' when ariaHideApp prop exist.
The second check is for the 'aria-hidden' applied on 'modal-target', based on the prop set to 'true' or 'false'.

Comment (StommePoes) [reply src/test/java/standAloneTests/ModalTest.java]: ah ok.

**Comment by umahaea** [comment]
all approved. merging it.

**Pull request merged**